api/serializers/author.py

```diff
@@ -98,14 +98,14 @@ class CompactAuthorSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Author
-        fields = ('id', 'displayname', 'host', 'url')
+        fields = ('id', 'displayname', 'host', 'url', 'image', )
```

Comment:
This is what we want but it doesn't exactly provide a the right representation of images. The image field is a file reference and that's what it ends up sending as JSON.

Need this to output the result of running `ImageSerializer` on that image field..
> image = ImageSerializer()

^ works and solves it but breaks a shit load of tests.